index-pl.html

```diff
@@ -616,54 +616,54 @@ <h2>Koszt ludzki</h2>
+      Co oznacza, że <b>im więcej osób nosi maski, tym ich korzyść dla społeczeństwa jest większa</b>.
     </p>
 
     <p>
-      If enough people wear masks, we can reach the bottom of the hill, where the chance of infection is zero.
-      This is how masks can end an epidemic.
-      But masks can only end an epidemic if enough people wear them.
+      Jeśli odpowiednia ilość osób nosi maski, możemy dotrzeć na sam dół krzywej, gdzie szanse zakażenia są zerowe.
+      W ten sposób maski mogą zakończyć epidemię.
+      Ale maski mogą zakończyć epidemię tylko gdy nosi je wystarczająco dużo osób.
     </p>
 
     <p>
-      You might wonder how many people have to wear masks to end an epidemic.
-      Well, that depends on how effective the masks are.
+      Możesz zastanawiać się jak wiele osób musi nosić maski, żeby zakończyć epidemię.
+      To zależy od tego jak bardzo są one skuteczne.
     </p>
 
     <p>
-      By playing with interactive above, you'll see that if masks were 50% effective, we'd need roughly three-quarters of the population to wear them to stop the spread of COVID-19.
-      But if masks were 75% effective, we'd only need half the population to wear them to stop the spread.
+      Przesuń wskaźniki na powyższym wykresie interaktywnym a zobaczysz, że przy skuteczności masek na poziomie 50%, około trzy czwarte populacji musi je nosić, żeby powstrzymać rozprzestrzenianie się COVID-19.
+      Przy skuteczności na poziomie 75%, musiałaby to być zaledwie połowa populacji.
     </p>
 
     <p>
-      <b>The more effective the mask, the faster we can descend the hill.</b>
-      That's why it's important to wear a mask that <a href="https://static1.squarespace.com/static/5e8126f89327941b9453eeef/t/5ea3b5859bc8f31a11f3deb5/1587787141808/2020-04-24_N95DECON_Face_Mask_Technical_Report_v1_final.pdf">tightly seals your mouth <i>and</i> nose</a>, and is made from an effective filtering material.
+      <b>Im skuteczniejsze maski, tym szybciej schodzimy z krzywej.</b>
+      Dlatego istotne jest, żeby nosić maski, które <a href="https://static1.squarespace.com/static/5e8126f89327941b9453eeef/t/5ea3b5859bc8f31a11f3deb5/1587787141808/2020-04-24_N95DECON_Face_Mask_Technical_Report_v1_final.pdf">szczelnie zakrywają usta <i>oraz</i> nos</a> i które wykonane są ze skutecznego materiału filtrującego.
     </p>
 
     <p>
-      We all want to get to the bottom of the hill and stop the spread of COVID-19.
-      But you can't get there by yourself.
-      Each person can only take a tiny step downwards.
+      Wszyscy chcemy zejść na sam dół krzywej zachorowań i zatrzymać rozprzestrzenianie się COVID-19.
+      Ale nie dotrzesz tam sam.
+      Pojdyncza osoba może zrobić jedynie mały krok w dół.
     </p>
 
-    <p>However, when many people take this small step, together, we take a giant leap down the hill.</p>
+    <p>Jednak jeśli wiele osób wspólnie zrobi ten mały krok, to wspólnie zrobimy wielki krok w dobrą stronę.</p>
```

Comment:
jednocześnie zrobi ten krok ?

Reply:
Nie podoba mi się "jednocześnie".
"Jednocześnie" wskazuje na to, że miałoby to się odbyć na raz, w tym samym czasie. Jednocześnie można machać ręką i dłubać w nosie, albo jednocześnie można przeskoczyć nad kałużą. A tu chodzi o to, żeby wiele osób robiło coś przez dłuższy okres czasu.